arviz/plots/backends/matplotlib/pairplot.py

```diff
@@ -162,14 +179,14 @@ def plot_pair(
         if point_estimate:
             pe_x = calculate_point_estimate(point_estimate, x)
             pe_y = calculate_point_estimate(point_estimate, y)
-            if diagonal:
+            if marginals:
                 ax_hist_x.axvline(pe_x, **point_estimate_kwargs)
                 ax_hist_y.axhline(pe_y, **point_estimate_kwargs)
 
             ax.axvline(pe_x, **point_estimate_kwargs)
             ax.axhline(pe_y, **point_estimate_kwargs)
 
-            ax.scatter(pe_x, pe_y, marker="s", s=figsize[0] + 50, **point_estimate_kwargs, zorder=4)
+            ax.scatter(pe_x, pe_y, zorder=4, **point_estimate_marker_kwargs)
```

Comment:
Even though `zorder` is not a commonly used argument, it should be passed as kwarg anyways.